### PR TITLE
Use gatsby directly to build the decks

### LINKED ISSUE #73: Invalid relative paths to js files
Currently the paths to the js assets is relative without the correct prefix (`/ausguck`). The result is that the html files cannot load their corresponding javascript and doesn't work as intended.

Issue at mdx-deck: https://github.com/jxnblk/mdx-deck/issues/535

### PR DESCRIPTION
**Pull request by Kampfheizung**
- fixes #73

### CODE REVIEW

**Review by Tobi-mmt — COMMENTED** [review]
I like the simplification  💪

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ dist
 node_modules
 ausguck/
 public
+.cache
```
[the remark this answers is not in the document]
Comment (Kampfheizung) [reply .gitignore]: 👍 done

File: package.json [hunk 2]
```diff
@@ -17,8 +16,12 @@
...
   },
   "devDependencies": {
-    "@mdx-deck/themes": "4.1.0",
-    "mdx-deck": "4.1.1",
+    "gatsby": "^2.24.63",
+    "gatsby-plugin-catch-links": "^2.3.12",
+    "gatsby-theme-mdx-deck": "^4.1.0",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-helmet": "^6.1.0",
```
[the remark this answers is not in the document]
Comment (Kampfheizung) [reply package.json]: done 👍

File: src/pages/index.mdx [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# All decks from farbenmeer
+
+- [Decks](/decks)
```
[the remark this answers is not in the document]
Comment (Kampfheizung) [reply src/pages/index.mdx:3]: Mhh we could list them and if we add a new deck we need to update the `index.mdx` – i'm okay with that 👍 i also didn't like the current routing `/ausguck` -> `/ausguck/decks` -> `/ausguck/decks/foo`

[the remark this answers is not in the document]
Comment (Tobi-mmt) [reply src/pages/index.mdx:3]: Is it possible to just move the `/decks` folder inside the `src/pages` folder?

[the remark this answers is not in the document]
Comment (Kampfheizung) [reply src/pages/index.mdx:3]: Sure somehow 😅 but that need a deep dive into the gatsby plugin `gatsby-theme-mdx-deck` or gatsby itself... feel free :D

[the remark this answers is not in the document]
Comment (Tobi-mmt) [reply src/pages/index.mdx:3]: or we change the config like that:

```
module.exports = {
  pathPrefix: '/ausguck',
  plugins: [
		'gatsby-plugin-catch-links',
		{
			resolve: 'gatsby-theme-mdx-deck',
			options: {
				contentPath: 'decks',
				basePath: '/'
			}
		}
	],
}
```

the the decks folder is routed to the root path and `/ausguck/cors` is working

[the remark this answers is not in the document]
Comment (Tobi-mmt) [reply src/pages/index.mdx:3]: We can also generate the index.mdx in the CI, as we did it before. So no manual adding of the decks here :)

**Review by Tobi-mmt — APPROVED** [review]

**Pull request merged**